cli/src/main/java/io/kestra/cli/commands/flows/FlowTestCommand.java

```diff
@@ -98,8 +98,8 @@ public Integer call() throws Exception {
             runnerUtils.runOne(
                 all.get(0),
                 (flow, execution) -> runnerUtils.typedInputs(flow, execution, inputs),
-                Duration.ofHours(1)
-            );
+                Duration.ofHours(1),
+                null);
 
```

Comment:
Why are you passing `null` here ? I think that the existing method without labels already exist?
This comment applies to all the existing test modifications where you just add a new `null` param.

Reply:
I've changed a wrong signature which caused superfluous `null`s.